src/objects/npo02__Households_Settings__c.object

```diff
@@ -22,4 +22,18 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>Seasonal_Addresses_Batch_Size__c</fullName>
+        <defaultValue>10</defaultValue>
+        <description>Number of records per batch to process when setting seasonal address as the mailing address.</description>
```

Comment:
@sam-knox , can you please review new field Description and Help Text as well as the PR "Changes" section? Thanks!